Mathport/Syntax/Translate/Basic.lean

```diff
@@ -10,6 +10,14 @@ import Mathport.Syntax.Translate.Attributes
 import Mathport.Syntax.Translate.Parser
 import Mathlib
 
+abbrev Lean.Syntax.Tactic := TSyntax `tactic
+abbrev Lean.Syntax.Conv := TSyntax `conv
+abbrev Lean.Syntax.Attr := TSyntax `attr
+abbrev Lean.Syntax.BracketedBinder := TSyntax ``Parser.Term.bracketedBinder
+abbrev Lean.Syntax.SimpleOrBracketedBinder := TSyntax [``Parser.Term.simpleBinder, ``Parser.Term.bracketedBinder]
+abbrev Lean.Syntax.EraseOrAttrInstance := TSyntax [``Parser.Command.eraseAttr, ``Parser.Term.attrInstance]
+abbrev Lean.Syntax.BinderIdent := TSyntax [identKind, ``Parser.Term.hole]
```

Comment:
The distinction still exists, maybe I should put in a comment.  The other one is called ```TSyntax ``binderIdent```.

> Otherwise this abbrev should be changed to `Syntax.Ident_`.

I'd rather not invent a new name.  We should only call the type `Syntax.Ident_` if it's `syntax ident_ := ...`.

Reply:
> I called the former `BinderIdent` and the latter `ident_` in the naming convention.

Missed this part.  Don't really like the made-up name, but we should stay consistent.

Reply:
Apparently the typed syntax feature produces the wrong antiquotation types for `ident <|> "_"`, namely `Ident`.  That's why I thought we needed an identifier there, so I added `trIdent__`.

I've cleaned this up now, following the naming convention with `Ident_` and `BinderIdent`.